host/videocore/videocore.go

```diff
@@ -94,10 +94,13 @@ var (
 	mu         sync.Mutex
 	mailbox    messager
 	mailboxErr error
+
+	// The 4 corresponds to the pointer size on the target platform. That means
```

Comment:
Yeah it probably happened to work because I never tested on x64 OS on a Raspberry Pi. Can you put size of int here? Ironically I have no idea how to do this (!) so just add `TODO(maruel):` there instead.

Reply:
I found a way using the `unsafe` package